src/Compilers/Shared/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -195,3 +198,5 @@ private string CreateUniqueDirectoryForAssembly()
         }
     }
 }
+
+#endif
```

Comment:
Please add the same `#else` pattern I recomended in the other file to proof against future changes.